tools/check_release_assets_uploaded.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import requests
+import sys
+import time
+
+def die(message):
+    print(message)
+    sys.exit(1)
+
+def get_assets_for_release(version_tag):
```

Comment:
can you add a comment on the return value.

Reply:
Done